### ISSUE TITLE
Support for Find Source Definition

### ISSUE THREAD

**Issue by johnsoncodehk** [vuejs/language-tools#1245]
Related: https://github.com/microsoft/TypeScript/pull/48264

**Comment by saminton** [comment]
Any news on this ? 

For anyone else that ends up here, here are the current workarounds I've found

The [vue-helper](https://marketplace.visualstudio.com/items?itemName=shenjiaolong.vue-helper) plugin does resolve this to some extent, but unfortunately the .d.ts file take priority over the plugin. Both are suggested in the list but the .d.ts file is always first. Maybe a temporary fix could be to disable the .d.ts definition files ?

I have tried adding `"volar.vueserver.noProjectReferences": true` and disabling the default VSCode typescript plugin but the results are always the same.

![Screenshot 2022-11-17 at 13 06 46](https://user-images.githubusercontent.com/15970757/202441942-b94eb791-59a1-4d8b-b008-d1754eee4079.png)

There is also the [Goto definition alias](https://marketplace.visualstudio.com/items?itemName=antfu.goto-alias) plugin which will first open the .d.ts file and then the component file which isn't ideal either.

An in-house solution would be much appreciated.
